docs/TemplateManagementCLI.md

```diff
@@ -0,0 +1,100 @@
+Template Management CLI is a tool to manage template files for FHIR Converter DotLiquid engine. This CLI tool manages all template files in [OCI image](https://github.com/opencontainers/image-spec) format and allows pushing and pulling templates with [Azure Container Registry](https://azure.microsoft.com/en-us/services/container-registry/).
```

Comment:
Let's give some context. 

"The $convert-data operation in the FHIR Server for Azure takes templateCollectionReference parameter, which refers to an [OCI image](https://github.com/opencontainers/image-spec) on Azure Container Registry (ACR). It is the image containing  Liquid templates to use for conversion. 

The Template Management CLI tool is mean to pull, push, and manage the templates on the ACR."